spec/requests/schools_request_spec.rb

```diff
@@ -0,0 +1,107 @@
+require 'rails_helper'
+
+RSpec.describe "Schools", type: :request do
+
+  describe "Creating a school" do
```

Comment:
The convention is to add the url endpoint as part of this description. It gives helpful info about the routes etc so this would be along the lines of: describe 'Creating a school - POST /schools'